fastly/base_fastly_service.go

```diff
@@ -15,7 +15,7 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
-var fastlyNoServiceFoundErr = errors.New("No matching Fastly Service found")
```

Comment:
go errors should have the format `err<Something>`.

go errors should not start with capital letter, hence you'll see a bunch of similar changes through out this PR.